classes/AddressFormat.php

```diff
@@ -236,11 +236,27 @@ public function checkFormatFields()
                     }
                 }
             }
+            $this->_checkRequiredFields($usedKeyList);
         }
 
         return (count($this->_errorFormatList)) ? false : true;
     }
 
+    /**
+     * Checks that all required fields exist in a given fields list.
+     * Fills _errorFormatList array in case of absence of a required field.
+     *
+     * @param array $fieldList
+     */
+    protected function _checkRequiredFields($fieldList)
```

Comment:
No reason for underscore in the function name.

```suggestion
    protected function checkRequiredFields($fieldList)
```

Reply:
done !